adoc/chapters/programming_interface.adoc

```diff
@@ -6528,18 +6528,24 @@ parameter: [code]#access_mode::read#, [code]#access_mode::write# and
 | Access target | Meaning
 | [code]#target::device#
     | Access a buffer from a <<sycl-kernel-function>> via device global memory.
+      Also used to get a native backend handle to the buffer from within a host
+      task.
 | [code]#target::host_task#
-    | Access a buffer from a <<host-task>>.
+    | Access a buffer's data on host from within a <<host-task>>.
 |====
 
-Programs which specify the access target as [code]#target::device# and then
-capture the [code]#accessor# in a <<host-task>> can only use the accessor for
-interoperability through the [code]#interop_handle#, any other uses result in
-undefined behavior.
+When an [code]#accessor# is used from within a <<sycl-kernel-function>>, the
+access target must be [code]#target::device#, otherwise the behavior is
+undefined.
```

Comment:
Should also be defined for `target::constant_buffer`.

Reply:
@hdelan: The WG discussed this today, and they want to list the deprecated access targets here also.  The WG mentioned `target::constant_buffer`, but there is also `target::local`.  I'd suggest saying something like:

> When an [code]#accessor# is used from within a \<\<sycl-kernel-function\>\>, the
access target must be [code]#target::device#, [code]#target::constant_buffer#, or [code]#target::local#; otherwise the behavior is undefined.  See sections XXX and YYY for a description of the deprecated [code]#target::constant_buffer# and [code]#target::local# targets.

Reply:
Thanks. Have added this in latest commit.

Reply:
Looks good to me.  Thanks!

Reply:
Great I will squash commits now

Reply:
I think there's no need.  We generally leave commits as they are in the PR.  When we merge a PR to the repo, we do a "merge commit", not "squash and merge".

Reply:
The commits do still exist somewhere, so it is always possible to change your mind. Either by looking at the commit hashes here or running `git reflog` on your side.

Reply:
I think this is a pretty self contained patch so I'm happy keeping it as a single commit